### PR TITLE
Basic commands for github and linux

### PR DESCRIPTION
**Pull request by kulkarni90**

### CODE REVIEW

File: project_devops/gitbasics.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+git config --global user.name "username"
+git config --global user.email "email"
+git init
```
Comment (david1412) [inline project_devops/gitbasics.txt:3]: Why did you wite this command

**Review by david1412 — COMMENTED** [review]
Please see my comments

File: project_devops/gitbasics.txt (hunk 1 above)
Comment (kulkarni90) [reply project_devops/gitbasics.txt:3]: To revise daily to get perfection in commands

Comment (david1412) [reply project_devops/gitbasics.txt:3]: Good.

**Review by david1412 — APPROVED** [review]
Look goog

**Review by nithindeva — APPROVED** [review]
looks good

**Pull request merged**